heroku/deploy-a-django-app.md

```diff
@@ -575,3 +579,31 @@ of data to use a larger Heroku Postgres plan for review apps. Unfortunately, the
 is not currently a way to set this type of configuration (and hence prevent these
 kinds of emails being sent for review apps) because [Heroku defaults to the cheapest plan for
 review app addons](https://help.heroku.com/SY28FR6H/why-aren-t-i-seeing-the-add-on-plan-specified-in-my-app-json-in-my-review-or-ci-app).
+
+### Help! My staging pipeline doesn't have a database!
+You might deploy a review app and everything works. Then you merge your code to master, which builds a new version to the staging pipeline. But for some reason, there is no database provisioned for staging.
+
+Did you have the `manifest` CLI plugin installed when you first created the Heroku pipeline? If not, then it won't provision the Postgres add-on. [See this step](#install-the-heroku-cli-with-the-manifest-plugin).
+
+Here's an example where the `manifest` plugin **was not installed** when creating an app:
```

Comment:
I think it would be more useful to include the expected output in the section where we create the apps, that way we can catch this error early.

Reply:
Good idea.

Reply:
Included it here too: c4c7a7a.